Svc/BufferLogger/test/ut/Health.cpp

```diff
@@ -23,7 +23,7 @@ namespace Svc {
       U32 key = 42;
 
       this->invoke_to_pingIn(0, key);
-      this->dispatchAll();
```

Comment:
This looks like a behavioral change. "all" becomes "one"

Reply:
Yeah, this change resulted from me trying to work around private access to the QueuedComponentBase class and raw component queue access. I decided that reworking that was complicated enough that it should be it's own pull request.

This change isn't required anymore and I can revert it, but I believe it's safe enough. As you can see, the test case only queues up a single port call so I don't think multiple calls to pingOut are necessary.